rollup.config.mjs

```diff
@@ -43,7 +43,7 @@ export default [
         presets: ['@babel/env', '@babel/typescript'],
         plugins: [
           '@babel/plugin-proposal-export-default-from',
-          '@babel/plugin-proposal-export-namespace-from',
+          '@babel/plugin-transform-export-namespace-from',
```

Comment:
Just out of curiosity, why was this needed?

Reply:
Suggested as the proposal has been merged and now it is avaiable as accepted change.